source/renderer/app/components/wallet/widgets/ClearButton.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { PopOver } from 'react-polymorph/lib/components/PopOver';
+import SVGInline from 'react-svg-inline';
+import closeIcon from '../../../assets/images/close-cross.inline.svg';
+import styles from './ClearButton.scss';
+
+interface Props {
+  label: string;
+  onClick: () => void;
+}
+
+export function ClearButton({ label, onClick }: Props) {
+  return (
+    <PopOver content={label} placement="top">
+      <button onClick={onClick} className={styles.component} tabIndex={-1}>
```

Comment:
Just a suggestion: You could parametrise the tabIndex value and set the `-1` as a default.

@edit We have very poor accessibility support anyway so disabling it permanently actually doesn't change our situation, so whatever :)